analysis/tests/src/expected/CompletionPipeChain.res.txt

```diff
@@ -0,0 +1,1970 @@
+Complete src/CompletionPipeChain.res 2:16
+posCursor:[2:16] posNoWhite:[2:15] Found expr:[2:11->0:-1]
+Completable: Cpath Value[arr]->
```

Comment:
Can you change the test so the results are 1 or 2 entries? (e.g. define your own little module)
What's going on is drowned in a sea of results.